perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/AttributeAlreadyAssignedException.java

```diff
@@ -4,6 +4,7 @@
 
 /**
  * Attribute is already assigned to some object. This exception raises when you assign (or add) attribute to some object which had the attribute assigned before.
+ * Currently, this exception is only used when assigning required attribute by the service.
```

Comment:
Do we really need to know for which purpose is this exception currently used? What is currently in the code? Do I need to remove this line if I will use this exception for other purpose than this one?